apps/google-analytics-4/frontend/src/App.tsx

```diff
@@ -23,6 +23,33 @@ const ComponentLocationSettings = {
 const App = () => {
   const sdk = useSDK();
 
+  const location = useMemo(() => {
+    for (const [location] of Object.entries(ComponentLocationSettings)) {
+      if (sdk.location.is(location)) {
+        return location;
+      }
+    }
+  }, [sdk]);
```

Comment:
- [ ] bug: this is returning `undefined` – resolve before merging

![image](https://user-images.githubusercontent.com/492573/222439590-f7adb48a-74a4-4a49-aeb0-2bb583528eef.png)

Reply:
resolved: 
![image](https://user-images.githubusercontent.com/492573/222474292-28c8cc80-d97a-4aec-b362-6e220421cdbe.png)